src/commands/notify.yml

```
@@ -57,8 +57,9 @@ parameters:
       default: true
   debug:
       description: |
-       View payload and response being sent to slack api.
-       Enable to view full payload being sent to slack and response being recieved from the API call.
+       View payload and response being sent to Slack API.
+       Enable to view full payload being sent to Slack and response being received from the API call.
+       Redacted content can be viewed by re-running the job with SSH and accessing the log files referenced in the job output.
```

Thank you for making grammar improvements, this makes the documentation much easier to read.

I'm not sure if it is in line with our other documentation, but it might be worth adding something about persistent environments.

```suggestion
       Redacted content can be viewed by re-running the job with SSH and accessing the log files referenced in the job output.
       When run in a persistent build environment such as CircleCI Runner, these debug log files may remain in the system's temporary filesystem indefinitely and accumulate over time.
```